tikv/raftstore/fsm_store.go

```diff
@@ -62,37 +62,44 @@ func (m *storeMeta) setRegion(region *metapb.Region, peer *Peer) {
 type mergeLock struct {
 }
 
-type PollContext struct {
+type GlobalContext struct {
```

Comment:
Maybe we could do more split on this?  `Global` still seems not quite direct :eyes:

Reply:
split by what?